### PR TITLE
Evil 31 add edit reservation modal

### PR DESCRIPTION
**Pull request by yoinker57**

### CODE REVIEW

File: app/(tabs)/reservation/index.tsx [hunk 1]
```diff
@@ -8,12 +8,12 @@ import { useState } from 'react'
...
 import InfoModal from '@/components/InfoModal'
 import Spinner from 'react-native-loading-spinner-overlay'
-import { SimpleReservation } from '@/constants/types'
+import { Reservation } from '@/constants/types'
 
 
 
 export default function Reservations() {
-    const [reservation, setReservation] = useState<SimpleReservation | null>(null)
+    const [reservation, setReservation] = useState<Reservation | null>(null)
```
Comment (irosikoni) [inline app/(tabs)/reservation/index.tsx]: I have an error here:
image

File: bun.lockb
Comment (irosikoni) [inline bun.lockb:1]: still worried that this file is updated after every bun install

File: components/reservation_components/EditReservationComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+                                    />
+                                    <Text style={styles.textStyle}>
+                                        Typ rezerwacji:
+                                    </Text>
+                                    <View style={styles.dropdownContainer}>
+                                        <Dropdown
+                                            data={reservationTypesData}
+                                            value={selectedType}
+                                            labelField={'label'}
+                                            valueField={'value'}
```
Comment (irosikoni) [inline components/reservation_components/EditReservationComponent.tsx:152]: image

**Review by irosikoni — CHANGES_REQUESTED** [review]

File: app/(tabs)/reservation/index.tsx (hunk 1 above)
Comment (yoinker57) [reply app/(tabs)/reservation/index.tsx]: done

File: components/reservation_components/EditReservationComponent.tsx (hunk 2 above)
Comment (yoinker57) [reply components/reservation_components/EditReservationComponent.tsx:152]: done

**Review by irosikoni — APPROVED** [review]
462650699_1215070449602625_3995049300764157572_n
Get rid of this warning.

462638470_3915827248654440_1499740856124662614_n

Change the size of this dropdown menu.

Also, after removing reservation, the modal should automatically close.

**Pull request merged**